pricing.js

```diff
@@ -1,36 +1,55 @@
-module.exports.calculateProductPrice = function (product, employee, coverageLevels) {
+function formatPrice(price) {
+  return parseInt(price * 100) / 100
+}
+
+
+function calcPrice(price, product) {
```

Comment:
This function is actually updating the price based on the employer contribution so the name should capture that. `calcPrice` is misleading.